extensions/pomodoro/src/pomodoro-menu-bar.tsx

```diff
@@ -50,10 +50,10 @@ export default function TogglePomodoroTimer() {
   }
 
   let icon;
-  icon = { source: { light: "tomato-light.png", dark: "tomato-dark.png" } };
+  icon = { source: { light: "tomato-light-0.png", dark: "tomato-dark-0.png" } };
```

Comment:
instead of different images for light and dark, you can use a [tint](https://developers.raycast.com/api-reference/user-interface/icons-and-images#properties)

```
icon = { source: 'tomato.png', tint: Color.PrimaryText }
```